components/console-backend-service/internal/domain/eventing/service.go

```diff
@@ -0,0 +1,103 @@
+package eventing
+
+import (
+	"fmt"
+
+	"knative.dev/pkg/apis"
+
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlschema"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/resource"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/tools/cache"
+	v1 "knative.dev/pkg/apis/duck/v1"
+)
+
+var triggersKind = "Trigger"
+
+var triggersGroupVersionResource = schema.GroupVersionResource{
+	Version:  v1alpha1.SchemeGroupVersion.Version,
+	Group:    v1alpha1.SchemeGroupVersion.Group,
+	Resource: "triggers",
+}
+
+var triggersSubscriberRefIndex = "subscriberRef"
+
+func triggersSubscriberRefIndexKey(namespace string, subscriberRef *v1.KReference) string {
+	if subscriberRef == nil {
+		return ""
+	}
+	return fmt.Sprintf("%s/%s/%s/%s/%s",
+		namespace,
+		subscriberRef.APIVersion,
+		subscriberRef.Kind,
+		subscriberRef.Name,
+		subscriberRef.Namespace)
+}
+
+var triggersSubscriberURIIndex = "subscriberURI"
+
+func triggersSubscriberRefURIKey(namespace string, uri *apis.URL) string {
```

Comment:
Same as above ^
I think the word `get` would clarify this is a function, not a variable.